docs/get-started.md

```diff
@@ -29,34 +29,36 @@ Usually, GitHub will enable GitHub Pages as soon as a `gh-pages` branch is detec
 
 1. Go to your repository settings page
 1. Go to the "Pages" sub-section on the left
-1. Under "Source", change "None" to `gh-pages`
-1. In the folder dropdown, select `/(root)`
-1. Skip the "Theme Chooser" option 
+1. Under "Source", change "None" to "Deploy from a branch"
+1. In the Branch dropdown, select `gh-pages` and `/(root)`
 1. Click on "Save"
 
-After saving, you will see confirmation text "Your site is ready to be published at...". For more information on enabling GitHub Pages, see the article on the GitHub website: [Configuring a publishing source for your GitHub Pages site](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site).
+After saving, you will see confirmation text "Your site is live at..." at the top of the page. For more information on enabling GitHub Pages, see the article on the GitHub website: [Configuring a publishing source for your GitHub Pages site](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site).
 
 If you don't want to have a publicly-available status website, you don't have to enable publishing. If that is the case, you can also choose to keep your repository private. It is **not** possible to publish a status website from a private repository without using an API proxy with authentication. See [#54](https://github.com/upptime/upptime/issues/54) and the [Configuration for `apiBaseUrl`](https://upptime.js.org/docs/configuration#custom-api-base-url) to learn how to set up a status website from your private repository.
 
 ### Add repository secrets
 
 All sensitive information required, such as API keys, are provided as environment variables. These are stored as GitHub repository secrets (see [Creating and storing encrypted secrets](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets)).
 
-To make commits and publish your website, Upptime requires a personal access token (PAT) with the `repo` and `workflow` scope, stored as a repository secret `GH_PAT`. You can create a personal access token by following these steps:
+To make commits and publish your website, Upptime requires a personal access token (PAT) with the Actions, Contents and Workflow read-write permissions, stored as a repository secret named `GH_PAT`. You can create a personal access token by following these steps:
 
 1. Click on your profile picture on the top-right corner and select "Settings"
 2. In the left sidebar, select "Developer settings"
-3. In the left sidebar, click "Personal access tokens"
+3. In the left sidebar, click "Personal access tokens" > "Fine-grained tokens"
```

Comment:
Like I mentioned in the PR description, I have changed this to discuss only the new PAT kind. I can add instructions for Legacy PATs (maybe in a `details` element?) but for now I decided to not do that to encourage safer options (the new PATs can be scoped to a single repo!).